justfile

```
@@ -0,0 +1,21 @@
+uv := `which uv`
+
+export PY_COLORS := "1"
+export PYTHONBREAKPOINT := "pdb.set_trace"
+
+uv_run := "uv run --frozen --extra dev"
+
```

Let's add a `help` recipe here so it's easy for folks to discover all the possible recipes:

```suggestion
[private]
default:
    @just help
```